rosbag2_samples/api_samples/bag_recorder_nodes/src/simple_bag_recorder.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021, Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <rclcpp/rclcpp.hpp>
+#include <std_msgs/msg/string.hpp>
```

Comment:
Since std_msgs is being used it should be added as a dependency in CMakeLists.txt and package.xml accordingly.